controllers/flinkcluster_converter.go

```diff
@@ -389,6 +401,11 @@ func getDesiredTaskManagerDeployment(
 		PeriodSeconds:       60,
 		FailureThreshold:    5,
 	}
+	var saVolume, saMount = convertGCPConfig(clusterSpec.GCPConfig)
+	if saVolume != nil && saMount != nil {
+		volumes = append(volumes, *saVolume)
+		volumeMounts = append(volumeMounts, *saMount)
+	}
```

Comment:
take out the podspec as you did for jobmanager and job to keep code style consistency?

Reply:
Done.